newIDE/electron-app/app/LocalFileDownloader.js

```diff
@@ -1,14 +1,38 @@
 const fs = require('fs');
 const axios = require('axios');
 const log = require('electron-log');
+const { session } = require('electron');
+
+const findGDevelopCloudCookieValue = async () => {
```

Comment:
should this function be in the `Project.js` service?

Reply:
Can't be unfortunately because we're in `electron-app`, so outside of the "React.js app". This is on the electron main process (running on Node.js) while the React.js app is bundled with webpack/create-react-app.